core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -198,15 +198,38 @@ private[spark] class ExecutorAllocationManager(
     t.start()
   }
 
+  def currentTargetNumExecutors(): Int =
```

Comment:
I would just call this `targetNumExecutors: Int` to be more concise, and add a javadoc to explain what it is